package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
```

Comment:
Shall we enable dependabot to track these?

Reply:
Most certainly yes, as well as the GH actions. Added a basic config (with daily frequency and no grouping, I don't think that'll be too noisy here.)